.prettierrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "printWidth": 84,
+    "parser": "flow",
+    "tabWidth": 4,
+    "semi": true,
+    "singleQuote": false,
+    "trailingComma": "all",
+    "bracketSpacing": false,
+    "arrowParens": "always"
```

Comment:
Many of these are prettier defaults, but I thought being explicit about what the settings were was better.